yoyodyne/train.py

```diff
@@ -211,6 +212,7 @@ def get_model_from_argparse_args(
         eval_metrics.add(args.patience_metric)
     # Please pass all arguments by keyword and keep in lexicographic order.
     return model_cls(
+        attention_context=args.hard_attention_context,
```

Comment:
Can you lexicographically order this (see comment on line 213) so `attention-context` comes after `arch`? 

(The reason is that it makes it easier to tell if all the necessary arguments are being passed. We have had bugs like that earlier.)